### DISCUSSION TITLE
Downloaded Replays being shown in the Local Leaderboard

### DISCUSSION THREAD

**Discussion by krivey** [ppy/osu#17694] [Q&A] [answered]
I was messing around in lazer while playing for fun. I tried downloading a replay from the Global Leaderboard, and I watched the replay. But when it finished, I saw the replay that I downloaded being shown in the Local Leaderboard. I got confused at first and thought maybe it was a bug, so I tried downloading another replay. And the results are the same.

Screenshots:
Local Leaderboard
![image](https://user-images.githubusercontent.com/83916269/162087901-56fa2359-6975-4cf9-8b74-046540438d31.png)

Global Leaderboard
![image](https://user-images.githubusercontent.com/83916269/162088066-8b7aeb79-62af-4d3f-a7b9-0ac9eb1be839.png)

I don't know if this is a bug or not, but I'm pretty sure this wasn't intended because it doesn't happen in stable and I also don't think it's supposed to do that.

Logs
[database.log](https://github.com/ppy/osu/files/8435636/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/8435637/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/8435638/network.log)
[performance.log](https://github.com/ppy/osu/files/8435640/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8435642/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8435643/updater.log)

**Comment by smoogipoo** [comment, marked as answer]
This is intentional.

**Comment by respektive** [comment]
>I don't know if this is a bug or not, but I'm pretty sure this wasn't intended because it doesn't happen in stable and I also don't think it's supposed to do that.

Just FYI, stable also saves them to local leaderboard when opening a replay file. It just doesn't happen when downloading one in game.
